DBSetup/createDB.sql

```diff
@@ -1,31 +1,40 @@
 create database VA_Trial;
 \c va_trial
+
+create table Patient (
+PatientID int NOT NULL,
+Deceased bool NOT NULL,
+FirstName text NOT NULL, 
+LastName text NOT NULL,
+Email text NOT NULL,
+PatientNotes text,
+PRIMARY KEY (PatientID)
+);
+
 create table SessionKeys (
 SessionKeyID text,
 UserID int, 
-CreatedTime bigint,
-LastUsedTime bigint,
-PRIMARY KEY (SessionKeyID)
+CreatedTime bigint, LastUsedTime bigint, PRIMARY KEY (SessionKeyID)
 );
 
 create table tfiSurvey ( 
 tfiSurveyID SERIAL, 
-PatientID int, 
-TFI_I int,
-TFI_Sc int, 
-TFI_C int, 
-TFI_SI int,
-TFI_A int,
-TFI_R int, 
-TFI_Q int, 
-TFI_E int, 
+PatientID int REFERENCES Patient(PatientID) ON DELETE CASCADE,
+TFI_I float,
+TFI_Sc float, 
+TFI_C float, 
+TFI_SI float,
+TFI_A float,
+TFI_R float, 
+TFI_Q float, 
+TFI_E float, 
 TFI_OverallScore int,
```

Comment:
The overall score should also be a float.
CALCULATION OF OVERALL TFI SCORE: (1) Sum all answers (maximum possible score = 250 at the maximum value of 10).
(2) Divide by 25 (yields the respondent's mean item score for all items having valid answers).
(3) Multiply by 10 (provides that respondent's overall TFI score within 0-100 range).

Reply:
Representing these values as `float` may be annoying for statistics later; numbers which cannot be represented exactly (anything not a sum of powers of two, eg 1/25) may differ from each other by 0.00000001 and therefore not be equal to one another in situations where you reach the same number by different formulae and have different rounding errors. The standard solution is to simply store as an integer which represents a constant multiple of the correct value.  Eg, you never store dollar amounts as a float; instead you store an integer number of cents. However this may be unnecessary if these numbers are never compared for equality, only greater than/less than, or may be a small enough issue to just let go.

Reply:
Valid points:
   @OkapalDominic How much work will it be to convert the front end to hold these variables as scaled integers? I'm willing to perform the conversion; however, we should make this decision **as soon as possible** and stick with it. It will be more difficult to change our minds later this week.

Reply:
Candice did say that two decimal places would be enough precision and from the queries asked for I believe they will mostly be used to find an average over a period of time.  So we could just easily store them as integers multiplied by 100, and convert when displaying them.

Reply:
So...integer or float?

Reply:
We will keep them as floats for now. We can make a ticket to change the whole system over to integers. but we can't change the back end without changing the front end at the same time. That change should have happened when Dominic made his PR for the front end.